pbm/backup/backup.go

```diff
@@ -252,7 +264,7 @@ func (b *Backup) run(bcp pbm.BackupCmd) (err error) {
 	}
 
 	oplog.SetTailingSpan(oplogTS, lwTS)
-	_, err = Upload(b.ctx, oplog, stg, bcp.Compression, rsMeta.OplogName)
+	_, err = Upload(b.ctx, oplog, stg, bcp.Compression, rsMeta.OplogName, -1)
```

Comment:
Is this leaving PBM at risk of failing if an oplog segment is > 100GB?
It's reasonable not to expect this i.m.o., but I would add a comment such as 'Not setting size parameter in Upload() because we don't expect an oplog span to ever be bigger than the default AWS max file size.'